ci/run_premerge_gpu.sh

```diff
@@ -75,24 +77,27 @@ verify_bundle() {
             python $(pwd)/ci/prepare_schema.py --l "$bundle_list"
         for bundle in $bundle_list;
         do
-            # Check if the bundle is "maisi_ct_generative", if so, set local environment (venv cannot work with xformers)
-            if [ "$bundle" == "maisi_ct_generative" ]; then
-                echo "Special handling for maisi_ct_generative bundle"
-                set_local_env requirements-dev.txt
-            else
-                init_venv requirements-dev.txt
```

Comment:
we should ensure the test env is clean for each bundle, otherwise CI cannot find missing dependencies and make have lib conflicts